.eslintrc.js

```diff
@@ -23,9 +23,6 @@ module.exports = {
         sourceType: 'module',
         ecmaVersion: 'latest',
       },
-    },
-    {
-      files: ['**/.eslintrc.{js,cjs,mjs}', '**/*.config.{js,cjs,mjs}'],
       env: {
```

Comment:
I needed the node env to apply to all `.mjs` file otherwise the lining broke because of unrecognized globals.